File: problemas/geometria-computacional/2840/2840.lua
entrada = {}
for numero in io.read():gmatch("%S+") do
  entrada[#entrada + 1] = tonumber(numero, 10)
end
r = entrada[1]
l = entrada[2]

pi = 3.1415
v = (4.0 * pi * r * r * r) / 3.0

print(string.format("%d", math.floor(l / v)))

File: problemas/geometria-computacional/2840/2840.kt
import java.util.*
import kotlin.math.floor

fun main(args: Array<String>) {
  val sc = Scanner(System.`in`)

  val r = sc.nextInt()
  val l = sc.nextInt()

  val pi = 3.1415
  val v = 4.0 * pi * r * r * r / 3.0

  println(String.format("%d", floor(l / v).toInt()))
}
